pkg/cli/admin/upgrade/upgrade.go

```diff
@@ -70,6 +70,9 @@ func New(f kcmdutil.Factory, streams genericclioptions.IOStreams) *cobra.Command
 			itself and report status that is available via "oc get clusterversion" and "oc describe
 			clusterversion".
 
+			Passing --to-multi-arch will upgrade the cluster from a single to a multi cluster
```

Comment:
nit: "multi cluster architecture" feels a bit weird.  Can we mirror [the docs][1] with "multi-architecture cluster"?  Something like:

> Passing `--to-multi-arch` will request a transition from a single-architecture cluster to a multi-architecture cluster.

[1]: https://docs.openshift.com/container-platform/4.11/post_installation_configuration/multi-architecture-configuration.html